### ISSUE TITLE
Is it possible to disable page reload completely ? 

### ISSUE THREAD

**Issue by centur** [KevinSheedy/chrome-tab-rotate#3]
Hi, Kevin. Is this possible to disable page reload for a given url completely ? 
e.g. we have a logging service (Seq to be specific ) that has built in ability to refresh the content, but to enable that - we have to click on a button. Currently Seq doesn't support that from url (https://github.com/datalust/seq-releases/issues/452). 
So our flow is a bit manual - we would set up all dashboard tabs and on a given tabs we need to click that autorefresh button. but tab rotator reloads those tabs on start (currently we are using revolver to rotate tabs because of this). Is it possible to adjust behaviour ot Tab Rotate so it would cater for such semi-manual scenario. 

- Tab Rotate loads on start, 
- Opens up all necessary tabs , 
- Then user interacts with pages in rotation
- Tab rotate continues to rotate these pages without reloading (to prevent losing user interaction results)

**Comment by KevinSheedy** [comment]
Hi @centur 
Disabling reload completely isn't possible at the moment. However, you can set the reload interval to a something like 999999999 and that should work for your use case.

"tabReloadIntervalSeconds": 999999999

I'll look into making it an option in a future release.

**Comment by centur** [comment]
Tried it - doesn't work as expected. Every time when I interact with the page (see above - we need to toggle that auto-refresh button on one of the tabs) - after a full cycle of tabs they're all being reloaded and all adjustments being lost. Would be happy to write a review once we really start using this extension. So far - we returned to Revolver and in-chrome setup.

PS: maybe an environment matters - for dashboards we are using Raspberry PI 2 with fully upgraded Raspbian Jesse with Chromium `Version 51.0.2704.91 Built on Ubuntu 14.04, running on Raspbian 8.0`
